projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-filtering.component.ts

```diff
@@ -522,19 +522,28 @@ export class IgxGridExcelStyleFilteringComponent extends BaseFilteringComponent
                 value: v
             }));
 
-            this.uniqueValues = this.column.sortStrategy.sort(items, 'value', SortingDirection.Asc, this.column.sortingIgnoreCase,
+            const isDescendingDirection = this.checkSortingDirection(items);
+
+            const sortingDirection = isDescendingDirection ? SortingDirection.Desc : SortingDirection.Asc;
+
+            this.uniqueValues = this.column.sortStrategy.sort(
```

Comment:
How can we be certain that everyone would want that? If this change is indeed necessary, we should probably think of a way to give them control on this, not to guess what should happen.